examples/sveltekit-houdini/.env.example

```
@@ -0,0 +1,3 @@
+# Refer to the README on how to fill these variables
+PUBLIC_GRAFBASE_API_URL=http://localhost:5173/graphql
```

By default, our CLI uses 4000 port: 
```suggestion
PUBLIC_GRAFBASE_API_URL=http://localhost:4000/graphql
```